src/BizHawk.Emulation.Cores/Consoles/Nintendo/NES/NES.Core.cs

```diff
@@ -1149,6 +1149,8 @@ public void WriteMemory(ushort addr, byte value)
 				uint flags = (uint)(MemoryCallbackFlags.CPUZero | MemoryCallbackFlags.AccessWrite | MemoryCallbackFlags.SizeByte);
 				MemoryCallbacks.CallMemoryCallbacks(addr, value, flags, "System Bus");
 			}
+			DB = value;
+

```

Comment:
Spacing, otherwise LGTM
```suggestion

			DB = value;
```